rules.py

```diff
@@ -1086,6 +1087,14 @@ def check(self, element):
     ValidateOcdidLowerCase
 ]
 
+def md5sum(filename, blocksize=65536):
```

Comment:
We'll want to use the SHA 512/256 algorithm. hashlib does support that, but only in Python3. So the choices are:

1) Require python3
2) Require the pycryptodomex library, and use its 512/256 support:
https://pycryptodome.readthedocs.io/en/latest/src/hash/sha512.html

Reply:
What's the reason for that requirement? Why go through the trouble of adding an additional dependency or requiring Python 3 if we don't need a cryptographically secure hash function?

Reply:
MD5 is trivially broken and has been since about 2008. SHA-1 has been considered broken since about 2010, and Google stopped accepting SHA-1 signed SSL certificates two years ago. SHA 512/256 is a human-readable SHA-2 checksum. It's just unfortunate that Python2 is lagging behind in having basic signature support.

Reply:
Ack. I still think it works fine for this use case, unless feed providers become adversarial :) And it has the advantage of being very short and also human readable. I'm not opposed to changing it if you feel strongly though. Lmk and I can take another look.

Reply:
Switched to the cryptography library, v2.5, which has native support for SHA-512/256. Resolving.